scripts/rollup.config.js

```diff
@@ -45,8 +45,8 @@ const primedPlugins = {
 };
 
 /* General Globals */
-const moduleName = 'ReactBrightcovePlayer';
-const pluginName = 'react-brightcove-player';
+const moduleName = 'ReactPlayerLoader';
+const pluginName = 'react-player-loader';
```

Comment:
Should we change this to `brightcove-react-player-loader`? I think it'd be good to have that in the dist files. It doesn't matter for modules because you'll import `'@brightcove/react-player-loader'`.